modules/swagger-core/src/main/java/io/swagger/v3/core/jackson/ModelResolver.java

```diff
@@ -88,6 +88,7 @@
 
 public class ModelResolver extends AbstractModelConverter implements ModelConverter {
     Logger LOGGER = LoggerFactory.getLogger(ModelResolver.class);
+    private static List<String> NOT_NULL_ANNOTATIONS = Arrays.asList("NotNull", "NonNull", "NotBlank", "NotEmpty");
```

Comment:
would expose it as public (lacking a more thorough refactor getting rid of the static `ParameterProcessor`), to allow some extensibility